angular-load/angular-load.d.ts

```diff
@@ -1,15 +1,15 @@
-// Type definitions for angular-load v0.4.1
-// Project: https://github.com/urish/angular-load
-// Definitions by: david-gang <https://github.com/david-gang>
-// Definitions: https://github.com/borisyankov/DefinitelyTyped
-
-/// <reference path="../angularjs/angular.d.ts" />
-
-declare module angular.load {
-
-    interface IAngularLoadService {
-        loadScript(url:string): ng.IPromise<any>;
-        loadCss(url:string): ng.IPromise<any>;
-    }
-
-}
+// Type definitions for angular-load v0.4.1
```

Comment:
This entire file is unchanged. I found that every time I rebased or otherwise worked with a branch this file would show as changed to git and I couldn't discard the changes.